package.json

```diff
@@ -16,11 +16,11 @@
     "prettier": "prettier --write \"./**/*.{md,jsx,json,html,css,js,yml}\"",
     "prettier-check": "prettier --check \"./**/*.{md,jsx,json,html,css,js,yml}\"",
     "test": "pnpm jest",
-    "jest": "cross-env NODE_ENV=test LOG_LEVEL=error FEED_URL_INTERVAL_MS=200 jest -c jest.config.js --",
+    "jest": "cross-env LOG_LEVEL=error jest -c jest.config.js --",
```

Comment:
Let's get rid of the `cross-env` dependency while we're doing this.  We can remove the `LOG_LEVEL=silent` and `LOG_LEVEL=error` bits, since we define this in `env.developmetn` and load that into tests.

Reply:
Will do. 
[env.development ](https://github.com/Seneca-CDOT/telescope/blob/79249355423cb327413bf515fd1916d5e3b399c0/config/env.development#L203) seems to have `LOG_LEVEL` set to `debug`, so I wasn't sure if setting it to `error` or `silent` was intended

Reply:
We may want to tweak this later, but I'd love to drop the dependency on `cross-env` and solve it another way if necessary.